automation/devops_automation_infra/tests/k8s_tests/test_product_install.py

```diff
@@ -0,0 +1,19 @@
+import logging
+import os
+
+from devops_automation_infra.k8s_plugins.rancher import Rancher
+from pytest_automation_infra.helpers import hardware_config
+
+
+@hardware_config(hardware={"host1": {"hardware_type": "vm", "base_image": "gravity_infra_230"}},
+                 grouping={"cluster1": {"hosts": ["host2"]}})
+def test_install_bt(base_config):
+    rancher = base_config.clusters.cluster1.Rancher
+    rancher.cli_login()
+    layer_name, version, catalog = "bettertomorrow-v2-data", "2.3.1-master", "online"
```

Comment:
this cannot be here .. we should not add anyvision specific stuff in this project

Reply:
yes.. this test should be in core-product repo or one of the bt repos?